packages/vite/src/node/utils.ts

```diff
@@ -58,8 +58,10 @@ interface DebuggerOptions {
   onlyWhenFocused?: boolean | string
 }
 
+type ViteDebugScope = `vite:${string}`
+
 export function createDebugger(
-  ns: string,
+  ns: ViteDebugScope,
```

Comment:
**suggestion:** Just a little _chore_ here, but is this `namespace`? Should we write it out?
... It's not nanoseconds :shrug: :laughing:

Reply:
done